src/TextField/TextField.jsx

```diff
@@ -370,6 +391,12 @@ const TextField = React.createClass({
     if (this.props.onKeyDown) this.props.onKeyDown(event);
   },
 
+  _handleInputKeyUp() {
+    this.setState({currentCharCount: this.getValue().length});
+    if (this.props.charCountMax >= this.getValue().length) this.setState({charCountError: false});
```

Comment:
@newoga  I guess I can do this by importing underline and label in to TextCharCount to manipulate the style. is it a good idea?